includes/class-sensei-analysis.php

```diff
@@ -29,7 +29,7 @@ class Sensei_Analysis {
 	 * @param string $file
 	 */
 	public function __construct( $file ) {
-		$this->name      = __( 'Analysis', 'sensei-lms' );
+		$this->name      = __( 'Reports', 'sensei-lms' );
 		$this->file      = $file;
 		$this->page_slug = 'sensei_analysis';
```

Comment:
Should we also change the page slug? It will break old links if we do so tho.

Reply:
From my perspective, it is okay to change the page slug: it makes things consistent, and we have changed them recently during navigation restructuring (from `/wp-admin/admin.php?page=sensei_analysis` to `/wp-admin/edit.php?post_type=course&page=sensei_analysis`).

Reply:
OK, it makes sense then. I've changed the slug in ae57044e4c0fcc10e77ae823ab03cdedd8631acb, which also changes the screen id. I've also noticed some issues with teacher permissions which will be addressed in a separate PR.